docker/files/amplify/run.sh.base

```diff
@@ -20,5 +20,7 @@ cp /colonyCDappBackend/scripts/amplify-meta.json.base /colonyCDapp/amplify/backe
 cp /colonyCDappBackend/scripts/amplify-meta.json.base /colonyCDapp/amplify/\#current-cloud-backend/amplify-meta.json
 
 # Liftoff!
+npm run amplify mock &
 
-npm run amplify mock
+# Enable codegen
+npx graphql-codegen --watch "/colonyCDapp/src/graphql/**/*.graphql"
```

Comment:
That was way more trickier than I thought.

Amplify is pretty quick to start sending HTTP 200s and it does it even before it creates the DB tables which results in codegen still failing. I tried to detect when the `/graphql` endpoint is ready (which itself wasn't straightforward, the endpoint returns 404 for GET requests and 401 for POST without the auth header) but that would still happen before the tables were created. 

At this point I think I ran out of sensible ideas on how to detect when Amplify is fully finished with its setup and the API is ready to be queried. For now, I added a 10s delay after waiting for Amplify with `wait-on` which does the trick, but it will likely fail on slower machines or when we add new models.

Reply:
Actually, I might be able to write a custom codegen script that retries fetching the schema when it fails: https://the-guild.dev/graphql/codegen/docs/advanced/programmatic-usage. 

I'll look into that tomorrow.